src/main/java/io/jenkins/plugins/cloudevents/CloudEventsConfig.java

```diff
@@ -0,0 +1,55 @@
+package io.jenkins.plugins.cloudevents;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import jenkins.model.GlobalConfiguration;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+
+import java.util.List;
+
+/**
+ * Jenkins global configuration.
+ */
+@Extension
+public class CloudEventsConfig extends GlobalConfiguration {
```

Comment:
Necessary to mention that this is Global Configuration in the class name. Rename the class to CloudEventsGlobalConfig ?